test/Garnet.test/RespTests.cs

```diff
@@ -1753,5 +1767,43 @@ public void AppendTest()
             Assert.IsTrue(time.Value.TotalSeconds > 0);
         }
 
+        #region Regression tests
+
+        #region #290 INCR DECR operation zero
+        [Test]
+        public void IncrAndDecrTest_290()
```

Comment:
Isn't @The fix for the #290 covered by [TestCase("key1", 0)] SingleDecr?
If so the test case is not needed.